wordle-web/src/components/GameBoard.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <v-row class="justify-center" dense v-for="(word, r) in game.guesses" :key="r">
     <v-col cols="auto" v-for="(letter, c) in word.letters" :key="`${r}-${c}`">
-      <LetterButton :letter="letter" @click="letterClick(letter)" />
+      <LetterButton rounded="0" :letter="letter" @click="letterClick(letter)" />
```

Comment:
```suggestion
      <LetterButton rounded="0" :letter="letter" />
```
GameBoard buttons probably shouldn't do anything. You can further make them visually "disabled" using a bit of css. It's possible to disable buttons using vuetify, but it applies some custom dimming/transparency styles that seem difficult to work around.

The css option: style="pointer-events: none"

This would also make letterClick() obsolete.